app/scripts/lib/encryptor.js

```diff
@@ -0,0 +1,149 @@
+var ethUtil = require('ethereumjs-util')
```

Comment:
* we dont need to do anything ethereum specific
* we shouldnt follow ethereums lead of inefficient encoding of binary as hex (prefer base64)
* this should be a seperate module, in fact we should likely use some off-the-shelf module